core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -41,25 +46,39 @@
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.URI;
-import java.net.UnknownHostException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.GuardedBy;
+import javax.naming.NamingEnumeration;
```

Comment:
Looks like it is absent.  Progaurd complains about refs to missing javax.naming classes, but not the com.sun ones.  I think if I check for both, it will properly disable itself.